cmd/kubelet/app/server.go

```diff
@@ -353,8 +353,14 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.KubeletDeps) (err error) {
 				if err != nil {
 					return err
 				}
+			} else {
+				glog.Warningf("failed to init kubelet configure sync : %v", err)
 			}
+		} else if cfgzErr != nil {
```

Comment:
This additional logging is not necessary. Errors during the configz setup are already logged by the `initConfigz` function. If the "unable to register configz" error is unclear, feel free to reword it.